Sources/NIO/EventLoopFuture.swift

```diff
@@ -921,9 +926,9 @@ extension EventLoopFuture {
     ///     - eventLoop: The `EventLoop` on which the new `EventLoopFuture` callbacks will fire.
     ///     - updateAccumulatingResult: The bifunction used to combine partialResults with new elements.
     /// - returns: A new `EventLoopFuture` with the combined value.
-    public static func reduce<U>(into initialResult: T, _ futures: [EventLoopFuture<U>], eventLoop: EventLoop, _ updateAccumulatingResult: @escaping (inout T, U) -> Void) -> EventLoopFuture<T> {
-        let p0 = eventLoop.makePromise(of: T.self)
-        var result: T = initialResult
+    public static func reduce<U>(into initialResult: Value, _ futures: [EventLoopFuture<U>], eventLoop: EventLoop, _ updateAccumulatingResult: @escaping (inout Value, U) -> Void) -> EventLoopFuture<Value> {
```

Comment:
Same question here about `U`. Actually, to avoid repeating this comment too many more times, let's assume this applies consistently through this file.

Reply:
thanks, renamed to `NewValue`, `OtherValue`, or `InputValue` depending on the context.